packages/styling/mixins/list-unstyled.scss

```diff
@@ -0,0 +1,7 @@
+@mixin list-unstyled {
+  padding-left: 0;
+  list-style: none;
+}
+.list-unstyled {
```

Comment:
Deze file staat in `mixins` maar heeft ook global styling. Dat kan niet, opknippen naar component en mixin file.

Reply:
Check